ibm_catalog.json

```diff
@@ -67,18 +85,22 @@
           "architecture": {
             "features": [
               {
-                "title": " Creates and configures an instance of watsonx.data",
-                "description": "This architecture creates and configures an instance of IBM watsonx.data. It accepts or creates a resource group."
+                "title": "Watsonx data instance",
+                "description": "Creates and configures an instance of watsonx.data."
+              },
+              {
+                "title": "Key Rings and Keys",
+                "description": "Creates Key Rings and Keys for either watsonx.data instances."
               }
             ],
             "diagrams": [
               {
                 "diagram": {
-                  "caption": "watsonx.data on IBM Cloud",
+                  "caption": "IBM Watsonx.data architecture",
```

Comment:
Watsonx -> wrong way to write. It should be `watsonx`
Previous caption was correct.